README.md

```diff
@@ -66,6 +66,9 @@ parameters:
         container_xml_path: var/cache/dev/srcApp_KernelDevDebugContainer.xml
         # or with Symfony 5+
         container_xml_path: var/cache/dev/App_KernelDevDebugContainer.xml
+    # If you're using PHP config files for Symfony 5.3+, you also need this for auto-loading of `Symfony\Config`:
+    scanDirectories:
+        - %currentWorkingDirectory%/var/cache/dev/Symfony/Config
```

Comment:
The `%currentWorkingDirectory%` part is not necessary.

Reply:
Thanks!